wsdl_docs.module

```diff
@@ -273,11 +273,19 @@ function wsdl_docs_create_wsclient_service_upload($name, $filename) {
  * @param object $service
  *   wsclient_service entity.
  * @param object $DOMDocument
+ * @throws \Exception|\WSClientException
  */
 function wsdl_docs_update_wsclient_service($service, $DOMDocument = null) {
   // Rebuild types and operations in service.
-  $endpoint = $service->endpoint();
-  $endpoint->initializeMetadata(TRUE);
+  try {
+    $endpoint = $service->endpoint();
+    $endpoint->initializeMetadata(TRUE);
+  }
+  // wsclient_soap.module function initializeMetadata() can throw an uncaught WSClientException exception
+  catch (WSClientException $e) {
+    throw $e;
```

Comment:
This code block doesn't actually do anything. You're catching the exception, but then re-throwing it, so PHP is just going to go back to trying to find a function that wants to handle it. If you didn't catch the exception here, the same thing happens.

Reply:
Got it - no need to re-throw the exception. 
https://www.php.net/manual/en/language.exceptions.php
"When an exception is thrown, code following the statement will not be executed, and PHP will attempt to find the first matching catch block."